commands/copilot_cli.py

```diff
@@ -148,7 +149,7 @@ def make_cloudformation(storage_config_file, output, overwrite):
             template = templates["svc"]["s3-policy"]
 
             for svc in storage_config.get("services", []):
-                path = Path(f"copilot/{svc}/addons/")
+                service_path = Path(f"copilot/{svc}/addons/")
```

Comment:
We tested this manually for now. It works.

I don't see any benefit in not fixing it as without this fix it will put all the addons after the first one that should be a service addon in the wrong place, e.g. `web/addons`.

Reply:
Ah OK, so it was overwriting `path` and affecting the subsequent env level addons?

That fix seems fine.